web/elm/src/Dashboard/Footer.elm

```diff
@@ -148,10 +181,20 @@ legend model =
                     , PipelineStatusSucceeded PipelineStatus.Running
                     ]
                 ++ legendSeparator model.screenSize
-                ++ [ toggleView model.highDensity ]
+                ++ [ toggleView (isHighDensity model) ]
         ]
 
 
+isHighDensity : { a | route : Routes.Route } -> Bool
```

Comment:
Errrrrgh. Duplication of the same function in Dashboard.Dashboard. Possibly this would make sense being moved to Routes and made a Routes.Route -> Bool function?